bin/logstash.lib.sh

```diff
@@ -47,6 +47,13 @@ setup_java() {
     JAVA_OPTS="$JAVA_OPTS -Xmx${LS_HEAP_SIZE}"
   fi
 
+  if [ "$LS_JMX_MONIT_PORT" ] ; then
```

Comment:
Your name is a lot better.

/purbon

On Friday, May 29, 2015, Jordan Sissel <notifications@github.com> wrote:

> In bin/logstash.lib.sh
> <https://github.com/elastic/logstash/pull/3320#discussion_r31351047>:
>
> > @@ -47,6 +47,13 @@ setup_java() {
> >      JAVA_OPTS="$JAVA_OPTS -Xmx${LS_HEAP_SIZE}"
> >    fi
> >
> > +  if [ "$LS_JMX_MONIT_PORT" ] ; then
>
> Want to just call this variable LS_JMX_PORT ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/3320/files#r31351047>.
>


-- 
Pere Urbon-Bayes
Software Engineer
https://www.elastic.co
https://twitter.com/purbon